tests/test_arraymanipulation.py

```diff
@@ -16,3 +17,78 @@ def test_asfarray(type, input):
     dpnp_res = dpnp.asfarray(input, type)
 
     numpy.testing.assert_array_equal(dpnp_res, np_res)
+
+
+class TestHstack:
+    def test_non_iterable(self):
+        numpy.testing.assert_raises(TypeError, hstack, 1)
+
+    def test_empty_input(self):
+        numpy.testing.assert_raises(ValueError, hstack, ())
+
+    def test_0D_array(self):
+        b = dpnp.array(2)
+        a = dpnp.array(1)
+        res = dpnp.hstack([a, b])
+        desired = dpnp.array([1, 2])
+        numpy.testing.assert_array_equal(res, desired)
+
+    def test_1D_array(self):
+        a = array([1])
```

Comment:
Could you please use `dpnp.` prefix as above to make it clear what exactly module used?